compiler/arser/include/arser/arser.h

```diff
@@ -261,6 +261,34 @@ class Arser
     return *_arg_map[arg_name];
   }
 
+  Argument &add_argument(const std::vector<std::string> &args)
+  {
+    _optional_arg_vec.emplace_back(args.at(0));
+    for (const auto &arg : args)
```

Comment:
this `arg` seems same level as `arg_name` from existing `add_argument`
how about using the same variable name for better understanding for new `add_argument` ?